test/test_symptom_prep.py

```diff
@@ -14,7 +14,7 @@ def prep():
 class TestSymptomPrep(object):
     def test_copy_variables(self, prep):
         headers = ['test1', 'test2', 'test3', 'test4']
-        row = dict(zip(headers, [2, 37, 0, 0]))
+        row = dict(list(zip(headers, [2, 37, 0, 0])))
```

Comment:
I really don't think this list() is necessary here and throughout this file where you have this pattern. I tested a bit on my own and got the same output with and without the list. The one thing that the linter I use suggests is setting `strict=True` inside the zip.